### PR TITLE
[plugin.video.heiseshowrss] 1.0

### PR DESCRIPTION
**Pull request by oscaropenness**
### Description
This plugin fetches the weekly podcast of the Heiseshow from its RSS-feed and displays it with description.

### Checklist:
- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

File: plugin.video.heiseshowrss/addon.xml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    <extension point="xbmc.python.pluginsource" library="addon.py">
+        <provides>video</provides>
+    </extension>
+    <extension point="xbmc.addon.metadata">
+        <summary lang="de_DE">#heiseshow Videopodcast</summary>
+        <description lang="de_DE">heise online spricht mit Gästen über aktuelle technische Entwicklungen und Netzpolitik.</description>
+        <summary lang="en_GB">#heiseshow Videopodcast</summary>
+        <description lang="en_GB">Watch the weekly news discussion #heiseshow from heise online</description>
+        <platform>all</platform>
+        <license>GNU GENERAL PUBLIC LICENSE. Version 3, June 2007</license>
```
Comment (basrieter) [inline plugin.video.heiseshowrss/addon.xml]: Could you use the SPDX identifier here https://spdx.org/licenses/

File: plugin.video.heiseshowrss/resources/lib/heiseshowrss.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        summary = item['description']    
+        
+        #Einige Folgen haben kein Thumbnailbild, die bekommen ein Standard-Bild verpasst
+        try:
+            thumb = item['image'].href
+        except:
+            thumb = "https://heise.cloudimg.io/bound/480x270/q75.png-lossy-75.webp-lossy-75.foil1/_www-heise-de_/ct/imgs/04/1/4/2/6/3/2/5/9cc02d4fe2a3a731.jpeg"
+    
+        #Baue fürs GUI ein Listenelement
+        list_item = xbmcgui.ListItem(label=title, label2=summary, thumbnailImage=thumb)
```
Comment (basrieter) [inline plugin.video.heiseshowrss/resources/lib/heiseshowrss.py]: Just to be Matrix ready, I would not use thumbnailimage. It is deprecated and you should use setArt() for that. See https://codedocs.xyz/AlwinEsch/kodi/group__python__xbmcgui__listitem.html

File: plugin.video.heiseshowrss/addon.xml (hunk 1 above)
Comment (oscaropenness) [reply plugin.video.heiseshowrss/addon.xml]: Done!

File: plugin.video.heiseshowrss/resources/lib/heiseshowrss.py (hunk 2 above)
Comment (oscaropenness) [reply plugin.video.heiseshowrss/resources/lib/heiseshowrss.py]: Done!

**Pull request merged**